packages/f-searchbox/src/components/formElements/FormSearchGeo.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <button
+        :class="$style['c-geo-locator']"
+        type="button"
+        @click="getSuggestions"
+        @keypress.enter="getSuggestions">
+        <span :class="$style['c-geo-locator-tip']">{{ copy.geoTip }}</span>
+
+        <geo-fill-icon
+            v-if="hasGeoSuggestionsResults" />
+
+        <geo-outline-icon
+            v-else />
+    </button>
+</template>
+
+<script>
+import { mapState, mapActions } from 'vuex';
+import { GeoOutlineIcon, GeoFillIcon } from '@justeat/f-vue-icons';
+
+export default {
+    components: {
+        GeoOutlineIcon,
+        GeoFillIcon
+    },
+
+    props: {
+        copy: {
+            type: Object,
+            default:  () => ({})
+        },
+
+        service: {
+            type: Object,
+            default: () => {},
```

Comment:
Same here, don't think the default is needed for a required prop

Reply:
Fixed.